CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ BUG FIXES:
 * agent: Avoid returning empty data on startup of a non-leader server [[GH-4554](https://github.com/hashicorp/consul/pull/4554)]
 * agent: Fixed a panic when serf_wan port was -1 but a reconnect_timeout_wan value was set. [[GH-4515](https://github.com/hashicorp/consul/issues/4515)]
 * catalog: Allow renaming nodes with IDs. [[GH-3974](https://github.com/hashicorp/consul/issues/3974)],[[GH-4413](https://github.com/hashicorp/consul/issues/4413)],[[GH-4415](https://github.com/hashicorp/consul/pull/4415)]
+* dns: Fixes a bug with the recursors to return the proper DNS response for the query issued.[[GH-4461](https://github.com/hashicorp/consul/pull/4461)]
```

Comment:
I think you could go with something that makes it clearer what the issue was and what is fixed, maybe a shorter version of this (from the PR):

> If a recursor returns a response like SERVFAIL or REFUSED the previous behavior was to return the DNS response, instead of moving onto the next recursor for getting the DNS query issued.